spec/index.adoc

```diff
@@ -560,7 +560,15 @@ In order to call a canister, the user makes a POST request to `/api/v2/canister/
 * `method_name` (`text`): Name of the canister method to call
 * `arg` (`blob`): Argument to pass to the canister method
 
-The HTTP response to this request has an empty body and HTTP status 202, or a HTTP error (4xx or 5xx). Paranoid agents should not trust this response, and use <<http-read-state,`read_state`>> to determine the status of the call.
+The HTTP response to this request can have the following responses:
+
+* 202 HTTP status with empty body. Implying the request was accepted by the IC for further processing. Users should use <<http-read-state,`read_state`>> to determine the status of the call.
+* 200 HTTP status with non-empty body. Implying an execution pre-processing error occurred. The body of the response contains more information about the IC specific error encountered. The body is a CBOR map with the following fields:
```

Comment:
Could we please list the cases explicitly? Currently, the reference implementation and its test suite does not handle this so I would like to know what to change there?

@rumenov

Reply:
I don't think this is reasonable. The [list is ever](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/http_endpoints/public/src/common.rs?L80) growing.

Reply:
> I see. Would they all become 200? Then it could make sense to list which 4xx/5xx are not transient.

I believe they all become 200. @dsarlis to confirm

Reply:
All these errors would become 200 (plus the cbor body that has the IC error details).

> Then it could make sense to list which 4xx/5xx are not transient.

I'm not sure I understood this part.

Reply:
> Then it could make sense to list which 4xx/5xx are not transient.

Given that all IC-specific errors would become 200, my comment is irrelevant (I wanted to list IC-specific errors - like canister not found - that still return 4xx/5xx instead of 200).